### DISCUSSION TITLE
How can I component-level import a css file from node_modules?

### DISCUSSION THREAD

**Discussion by willwill96** [vercel/next.js#16412] [Help] [answered]
In my project, we use Bootstrap on one of our pages. However, on every other page, we want to maintain our own custom styling. Right now, we are importing the `bootstrap.min.css` file in our `_app.js` file. The problem with this approach is that we import the bootstrap styles throughout the entire application, rather than on the sole page where we want it.

I have looked at the [Component-Level CSS docs](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css) and it seems like this could be adapted to how i want this to operate. 

My line of thinking is that I can import the bootstrap css file, and then re-export it as a `module.css`. My approach right now is using a `CopyWebpackPlugin` to rename the `boostrap.min.css` file to a `module.css` file. However I'm running into issues because bootstrap has selectors that are not "pure" when I import the `module.css` file.

![image](https://user-images.githubusercontent.com/10997562/90816403-6e0fad80-e2e9-11ea-9338-5374bc16908f.png)

Does anyone have any suggestions for how to get around this? Or maybe another approach that I haven't considered to accomplish this?

**Comment by jamesmosier** [comment, marked as answer]
There is an [RFC](https://github.com/vercel/next.js/discussions/15208) that relates to this as well as an issue that is similar https://github.com/vercel/next.js/issues/13282 (and also a discussion that links a few other issues https://github.com/vercel/next.js/discussions/15577#discussioncomment-43285)

Unfortunately just renaming it to be a `module.css` won't work (as you saw first hand). I haven't seen any other ways around this problem.
